deployment/v3/external/captcha/install.sh

```
@@ -34,7 +34,7 @@ function secret_setup() {
   while true; do
     read -p "Do you want to continue configuring Captcha secrets for admin ? (y/n) : " ans
       if [ "$ans" ='Y' ] || [ "$ans" = 'y' ]; then
```

Shellcheck (0.11.0)

[error] You are missing a required space here.

(SC1035)